internal/tags/framework.go

```diff
@@ -1,23 +1,23 @@
 package tags
 
 import (
-	"github.com/hashicorp/terraform-plugin-framework/tfsdk"
+	"github.com/hashicorp/terraform-plugin-framework/resource/schema"
```

Comment:
Note for our future selves: We will probably need data source-specific variants of these now that there are separate  `resource/schema` and `datasource/schema` packages.